<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminRateService</class>
 <widget class="QWidget" name="AdminRateService">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="tbRate">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>301</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="tbService">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>300</y>
     <width>301</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnApplyRate">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Rate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddService">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>390</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Add service</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lwRate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lwService">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>321</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbResponse">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>731</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
